improver/lapse_rate.py

```diff
@@ -326,12 +326,43 @@ def _create_heightdiff_mask(self, all_orog_subsections):
 
         height_diff = np.subtract(self.all_orog_subsections, central_points)
         height_diff = np.absolute(height_diff)
-
-        height_diff_mask = np.where(height_diff >= self.max_height_diff, True,
+        height_diff_mask = np.where(height_diff >=
+                                    self.max_height_diff, True,
                                     False)
 
         return height_diff_mask
 
+    def _generate_heightdiff_mask(self, all_orog_subsections):
+        """
+        Function to create a mask for any neighbouring points where the height
+        difference from the central point is greater than max_height_diff.
+
+        Slice through the orography subsection array to remove central
+        points.
+        Extracts the height value of each central point and masks out the
+        neighbouring points where their height difference is greater than
+        the maximum.
+
+        Args:
+            all_orog_subsections(2D numpy.ndarray):
```

Comment:
The argument type in round brackets should only contain the Python object type in order to make this understandable by type checker tools. Suggest moving the "2D" into the text description of the argument on the next line.

Reply:
Okay, I'll change it now